src/include/user/crossover.h

```diff
@@ -10,9 +10,7 @@
 
 #include <stdint.h>
 #include <user/eq.h>
-
-/* Maximum Number of sinks allowed in config */
-#define SOF_CROSSOVER_MAX_STREAMS 4
```

Comment:
Hmm, this is a bit problematic. The "user/crossover.h" header should not have dependency to implementation parts, it should be independent description of the parameter interface (like user/eq.h).

Reply:
it was referenced by multiband DRC user header file with:
/* Maximum number of frequency band for Multiband DRC */
#define SOF_MULTIBAND_DRC_MAX_BANDS SOF_CROSSOVER_MAX_STREAMS